doc/sphinx/how-to/how-to-enable-graphics-core22-on-a-device.md

```diff
@@ -0,0 +1,455 @@
+# How to enable graphics-core22 on a device
+
+
+This document will guide you through the process of assembling a 
+`graphics-core22` snap. With this snap, your device will be ready
+to run graphical snaps like `ubuntu-frame` or any other *Mir*-based
+compositor in a snapped environment.
+
+To this end, we will accomplish the following in order:
+- How to assemble a `graphics-core22` provider snap
+- How to test a `graphics-core22` provider snap
+- How to test the confinement of your *Mir*-based snap that relies on the provider snap
+
+## Prerequisites
+Before we get started, let's make sure that we have everything that we require.
+
+### 1. Development Board
+For starters, make sure that you have some sort of development board on hand.
+
+### 2. GPU Support Check
+Next, make sure that this board runs a `snapd`-enabled Ubuntu image with
+a kernel that includes GPU support. To do this, we'll run a series of checks.
+
+First, you should ensure that *at least* `/dev/dri/card0` exists, but you 
+may have more depending on your setup. You may check this from the command line.
+
+Next, install [drm_info](https://gitlab.freedesktop.org/emersion/drm_info). The
+output of `drm_info` will look something like the following:
```

Comment:
This is available in the `graphics-test-tools` snap, which *can* be installed before having a graphics-core22 provider (it'll pull in the mesa provider)